astro-iris/{{ cookiecutter.repo_name }}/src/requirements.txt

```diff
@@ -1,14 +1,14 @@
-black==v19.10b0
+black==21.5b1
 flake8>=3.7.9, <4.0
 ipython~=7.0
-isort>=4.3.21, <5.0
+isort>=4.3.21, <5.9
 jupyter~=1.0
 jupyter_client>=5.1, <7.0
-jupyterlab==0.31.1
+jupyterlab>=0.31.1, <3.1
```

Comment:
Do we actually want this to have such a low bound, rather than going for something like `jupyterlab~=3.0`?

Reply:
The reasons why I decided to keep bounds as wide as possible are:
1. That's the way [the ticket](https://jira.quantumblack.com/browse/KED-2540) is described
2. A conversation I had with @lorenabalan when I was working on the requirements for python 3.9
3. The fact that the lower end of the spectrum is atm guaranteed to pass the tests and hence supported

Reply:
Cool, makes sense. Thanks for the explanation 👍

Reply:
FYI @AntonyMilneQB in the end we're going for `jupyterlab~=3.0` as you suggested (people seem to prefer that one) :)